### PR TITLE
[MRG] Integer oversampling value

### PR DESCRIPTION
**Pull request by bthirion**
Addresses #103

### CODE REVIEW

**Comment by kchawla-pi** [comment]
It would be helpful to expand in the initial comment what is being done here. I don't think the PR in it's current state addresses #103 , it sets up the stage for it by addressing https://github.com/nistats/nistats/issues/103#issuecomment-531032085

File: nistats/hemodynamic_models.py [hunk 1]
```diff
@@ -482,16 +482,22 @@ def compute_regressor(exp_condition, hrf_model, frame_times, con_id='cond',
...
+        finite impulse response basis, a set of delayed dirac models.
     
     It is expected that spm standard and Glover model would not yield
     large differences in most cases.
 
     In case of glover and spm models, the derived regressors are
     orthogonalized wrt the main one.
     """
+    # fir_delays should be integers
+    if fir_delays:
```
Comment (bthirion) [inline nistats/hemodynamic_models.py]: Seems to trigger an error in Travis. I think it should be 
```suggestion
    if fir_delays is not None:
```

**Comment by kchawla-pi** [comment]
@bthirion If you address https://github.com/nistats/nistats/pull/412#issuecomment-559765063 that will be helpful in determining what else is to be done in this or after this PR.

**Comment by bthirion** [comment]
I propose to further impose that fir_delays have integer values only, which would solve the initial issue raise in #103 if I'm not mistaken.

File: nistats/hemodynamic_models.py [hunk 2]
```diff
@@ -482,16 +482,22 @@ def compute_regressor(exp_condition, hrf_model, frame_times, con_id='cond',
...
+    'fir': list or array,
+        finite impulse response basis, a set of delayed dirac models.
     
     It is expected that spm standard and Glover model would not yield
     large differences in most cases.
 
     In case of glover and spm models, the derived regressors are
     orthogonalized wrt the main one.
     """
+    # fir_delays should be integers
```
Comment (kchawla-pi) [inline nistats/hemodynamic_models.py]: Unnecessary comment.

File: nistats/hemodynamic_models.py [hunk 3]
```diff
@@ -482,16 +482,22 @@ def compute_regressor(exp_condition, hrf_model, frame_times, con_id='cond',
...
+    if fir_delays is not None:
+        fir_delays = [int(x) for x in fir_delays]
+
+    # oversampling should be an integer
```
Comment (kchawla-pi) [inline nistats/hemodynamic_models.py]: Unnecessary comment.

**Comment by kchawla-pi** [comment]
Pending:
 - The 2 unnecessary comments.
 - Update whats_new under new version

**Pull request merged**